From 3958033eda359821c288fbd7dee6226ad115f88f Mon Sep 17 00:00:00 2001
From: Gamer64ytb <76565986+Gamer64ytb@users.noreply.github.com>
Date: Sun, 28 Apr 2024 13:04:31 +0200
Subject: [PATCH] Fix again

---
 src/common/settings.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/common/settings.h b/src/common/settings.h
index b38bf83f8..8b51bf47b 100644
--- a/src/common/settings.h
+++ b/src/common/settings.h
@@ -511,7 +511,8 @@ struct Values {
 
     SwitchableSetting<bool> filter_mode{true, "filter_mode"};
     SwitchableSetting<std::string> pp_shader_name{"none (builtin)", "pp_shader_name"};
-    SwitchableSetting<std::string> anaglyph_shader_name{"rendepth (builtin)", "anaglyph_shader_name"};
+    SwitchableSetting<std::string> anaglyph_shader_name{"rendepth (builtin)",
+                                                        "anaglyph_shader_name"};
 
     SwitchableSetting<bool> dump_textures{false, "dump_textures"};
     SwitchableSetting<bool> custom_textures{false, "custom_textures"};
